README.md

```diff
@@ -1,49 +1,47 @@
 # CPAN.org content repository
 
-This repository is for maintaining the non-source-code, non-archive
-content on cpan.org.  Discussion related to this should be on the
-[cpan-workers mailing list][cpan-workers].
+This repository is for maintaining the non-source-code, non-archive content on [cpan.org](https://www.cpan.org).
```

Comment:
Please maintain 80 character column limit.

Reply:
Also below.

Reply:
> Please maintain 80 character column limit.

Just wondering, why maintaining 80 column limit is important? 🤔

Reply:
It's more compatible with the code change management tools (git and surrounding tooling). For example it makes diffs generally easier to read.